src/panels/config/zha/zha-group-binding.ts

```diff
@@ -0,0 +1,334 @@
+                    "ui.panel.config.zha.group_binding.group_picker_help"
+                  )}
+                </div>
+              `
+            : ""}
+          <div class="command-picker">
+            <zha-clusters-data-table
+              .hass=${this.hass}
+              .narrow=${this.narrow}
+              .clusters=${this._clusters}
+              @selection-changed=${this._handleClusterSelectionChanged}
+              class="menu"
+            ></zha-clusters-data-table>
+          </div>
+          ${this._showHelp
+            ? html`
+                <div class="helpText">
+                  ${this.hass!.localize(
+                    "ui.panel.config.zha.group_binding.cluster_selection_help"
+                  )}
+                </div>
+              `
+            : ""}
+          <div class="card-actions">
+            <mwc-button
+              @click="${this._onBindGroupClick}"
+              .disabled="${!(
+                this._groupToBind &&
+                this._clustersToBind &&
+                this._clustersToBind?.length > 0 &&
+                this.selectedDevice
+              )}"
+              >${this.hass!.localize(
+                "ui.panel.config.zha.group_binding.bind_button_label"
+              )}</mwc-button
+            >
+            ${this._showHelp
+              ? html`
+                  <div class="helpText">
+                    ${this.hass!.localize(
+                      "ui.panel.config.zha.group_binding.bind_button_help"
+                    )}
+                  </div>
+                `
+              : ""}
+            <mwc-button
+              @click="${this._onUnbindGroupClick}"
+              .disabled="${!(
+                this._groupToBind &&
+                this._clustersToBind &&
+                this._clustersToBind?.length > 0 &&
+                this.selectedDevice
+              )}"
+              >${this.hass!.localize(
+                "ui.panel.config.zha.group_binding.unbind_button_label"
+              )}</mwc-button
+            >
+            ${this._showHelp
+              ? html`
+                  <div class="helpText">
+                    ${this.hass!.localize(
+                      "ui.panel.config.zha.group_binding.unbind_button_help"
+                    )}
+                  </div>
+                `
+              : ""}
+          </div>
+        </ha-card>
+      </ha-config-section>
+    `;
+  }
+
+  private _bindTargetIndexChanged(event: ItemSelectedEvent): void {
+    this._bindTargetIndex = event.target!.selected;
+    this._groupToBind =
+      this._bindTargetIndex === -1
+        ? undefined
+        : this.groups[this._bindTargetIndex];
+  }
+
+  private _onHelpTap(): void {
+    this._showHelp = !this._showHelp;
+  }
+
+  private async _onBindGroupClick(): Promise<void> {
+    if (
+      this.hass &&
+      this._groupToBind &&
+      this._clustersToBind &&
+      this._clustersToBind?.length > 0 &&
+      this.selectedDevice
```

Comment:
Since you have this same logic 3 times, might be better to create a getter
```ts
private get _canBind(): boolean {
    return this._groupToBind &&
      this._clustersToBind &&
      this._clustersToBind?.length > 0 &&
      this.selectedDevice;
}